Lombiq.NodeJs.Extensions.Samples.NuGet/Assets/Styles/sample.scss

```diff
@@ -0,0 +1,11 @@
+.banner {
```

Comment:
Changing this file, or any other SCSS file in the other Samples project or in Privacy, doesn't update the CSS file on build, you need to executed an explicit build from the Task Runner Explorer.

Reply:
I'll have to investigate that. I've tried many things and don't recall this problem.

Reply:
I've found the problem and fixed it 😀